modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/low/level/search/options/portlet/shared/search/LowLevelSearchOptionsPortletSharedSearchContributor.java

```diff
@@ -56,31 +54,28 @@ public void contribute(
 					portletSharedSearchSettings.
 						getPortletPreferencesOptional());
 
-		Optional<String> connectionIdOptional =
-			lowLevelSearchOptionsPortletPreferences.getConnectionIdOptional();
-
 		SearchRequestBuilder searchRequestBuilder =
 			portletSharedSearchSettings.getFederatedSearchRequestBuilder(
 				lowLevelSearchOptionsPortletPreferences.
 					getFederatedSearchKey());
 
 		searchRequestBuilder.connectionId(
-			connectionIdOptional.orElse(null)
+			lowLevelSearchOptionsPortletPreferences.getConnectionIdString()
```

Comment:
no longer converting this to null if blank since all usages use Validator.isBlank() to check for null and blank